docker/nginx/libs/skynet/account.lua

```diff
@@ -58,7 +66,8 @@ function _M.get_account_limits()
         end
     end
 
-    return cjson.decode(ngx.var.account_limits)
+    curr_limits = cjson.decode(ngx.var.account_limits)
```

Comment:
it seems dangerous and counterintuitive to cache the value inside of this function,
I think it's more explicit if the result would be cached and we simply don't call this
function when we already have a cached result, seems like foot-gun territory to me

Reply:
I disagree with this assessment. Caching the data close to its use is a valid choice, especially if you don't use the cached value outside of the function. This becomes a part of the internal implementation of the function.